### ISSUE TITLE
Accept array on style props

### ISSUE THREAD

**Issue by lucasbento** [vincee48/react-native-checkbox-field#6]
I built a wrapper around the package to easily change it if I have to, something like:

```jsx
class MyCheckbox extends Component {
  render() {
    const { value, onChange, containerStyle } = this.props;

    return (
      <CheckboxField
        onSelect={onChange}
        selected={value}
        containerStyle={[styles.checkboxContainerStyle, containerStyle]}
        checkboxStyle={styles.checkboxStyle}
      >
        {value ? <Icon style={styles.checkColor}>check</Icon> : null}
      </CheckboxField>
    );
  }
}
```

I want to use a different `containerStyle` if it's provided through a prop, but it throws a warning since the passed style is not an object but an array.

**Comment by vincee48** [comment]
I have just published an update to 1.1.2 to allow array prop type for all style props. This should now support arrays passed into the styles and no longer throw the warning. Please update your package from 1.1.1 to 1.1.2 to pick up the latest changes.

**Comment by lucasbento** [comment]
Awesome, thanks @vincee48!
